lxqt-config-appearance/configothertoolkits.cpp

```diff
@@ -61,20 +64,39 @@ gtk-menu-images = %4
 gtk-button-images = %4
 gtk-toolbar-style = %5
 gtk-cursor-theme-name = %6
+gtk-cursor-theme-size = %7
 )GTK3_CONFIG";
 
 static const char *XSETTINGS_CONFIG = R"XSETTINGS_CONFIG(
 # Created by lxqt-config-appearance (DO NOT EDIT!)
 Net/IconThemeName "%2"
 Net/ThemeName "%1"
 Gtk/FontName "%3"
-Gtk/MenuImages %4
-Gtk/ButtonImages %4
-Gtk/ToolbarStyle "%5"
+# Gtk/MenuImages %4
+# Gtk/ButtonImages %4
+# Gtk/ToolbarStyle "%5"
 Gtk/CursorThemeName "%6"
+Gtk/CursorThemeSize %7
 )XSETTINGS_CONFIG";
 
-ConfigOtherToolKits::ConfigOtherToolKits(LXQt::Settings *settings,  LXQt::Settings *configAppearanceSettings, QObject *parent) : QObject(parent)
+static const char *GTK3_GSETTINGS = R"GTK3_GSETTINGS(
+set org.gnome.desktop.interface icon-theme "%2"
+set org.gnome.desktop.interface gtk-theme "%1"
+set org.gnome.desktop.interface font-name "%3"
+# set org.gnome.settings-daemon.plugins.xsettings overrides "{'Gtk/ButtonImages': <%4>, 'Gtk/MenuImages': <%4>}"
+# set org.gnome.desktop.interface toolbar-style "%5"
+set org.gnome.desktop.interface cursor-theme "%6"
+set org.gnome.desktop.interface cursor-size "%7"
+)GTK3_GSETTINGS";
+
+/*
+ * Button and menu images are deprecated in GTK 3.10:
+ *    https://developer.gnome.org/gtk3/stable/GtkSettings.html#GtkSettings--gtk-menu-images
+ * Toolbar-style is also deprecated.
+ * GTK just doesn't intend to support image menu items. They're considered bad practice in modern UI. 
+ */
+
+ConfigOtherToolKits::ConfigOtherToolKits(LXQt::Settings *settings,  LXQt::Settings *configAppearanceSettings, QWidget *parent) : QObject(parent)
```

Comment:
Would you please tell why you changed `QObject *parent` to `QWidget *parent`?

Reply:
Sorry. I changed but I don't remember why. QObject is better than QWidget here. I will restore QObject.